docs/resources/rule_attack_rechecker_rewrite.md

```diff
@@ -27,9 +27,9 @@ resource "wallarm_rule_attack_rechecker_rewrite" "default_rewrite" {
 
 ## Argument Reference
 
-* `rules` - (Required) A list of new values for parameters specified in `point`.
-* `action` - (Optional) Rule conditions. Possible attributes are described below.
-* `point` - (Required) Request parts to apply the rules to. The full list of possible values is available in the [Wallarm official documentation](https://docs.wallarm.com/user-guides/rules/request-processing/#identifying-and-parsing-the-request-parts).
+* `rules` - (**required**) a list of new values for parameters specified in `point`.
+* `action` - (optional) rule conditions. Possible attributes are described below.
+* `point` - (**required**) request parts to apply the rules to. The full list of possible values is available in the [Wallarm official documentation](https://docs.wallarm.com/user-guides/rules/request-processing/#identifying-and-parsing-the-request-parts).
```

Comment:
please mention that point here is for the parameters that should be changed before attack verification

Reply:
Did not understand that, let's discuss.